controller/KassiererController.cs

```diff
@@ -0,0 +1,114 @@
+using ZooAPI.model;
+
+namespace ZooAPI.controller;
+
+public class KassiererController
+{
+    
+    private static Dictionary<string,int> StringToDic(string input)
+    {
+        input = input.Replace("{", "");
+        input = input.Replace("}", "");
+        input = input.Replace("\"", "");
+        input = input.Replace("\n", "");
+        input = input.Replace(" ", "");
+        var valuePare = input.Split(',');
+        
+        Console.WriteLine("---");
+        Console.WriteLine(valuePare[1]);
+        Console.WriteLine("---");
+        
+        
+        var dic = new Dictionary<string,int>();
+        
+        
+        for (int i = 0; i < valuePare.Length; i++)
+        {
+            var pare = valuePare[i].Split(":");
+            Console.WriteLine(pare[0]);
+            dic.Add(pare[0], int.Parse(pare[1]));
+        }
+
+        
+
+        return dic;
+    }
+    
+
+    private static List<double> getPrices(Dictionary<string,int> dic)
+    {
+        double adultPrice = 8.50;
+        double childPrice = 8.50;
```

Comment:
It is convenient to store the ticket price in the db. It then is at a local place for every application to use.